### PR TITLE
Hanze development

### PR DESCRIPTION
**Pull request by HantzeWANG**
week 3

### CODE REVIEW

File: portfolio/src/main/java/com/google/sps/servlets/HelloWorldServlet.java [hunk 1]
```diff
@@ -10,9 +15,27 @@
...
+    String s2 = "Hola";
+    String s3 = "Guten tag";
+    strings.add(s1);
+    strings.add(s2);
+    strings.add(s3);
+    
+    Gson gson = new Gson();
+    String json = gson.toJson(strings);
+
+    // response.setContentType("text/html;");
```
Comment (EvanJuV) [inline portfolio/src/main/java/com/google/sps/servlets/HelloWorldServlet.java]: Please remove these comments.

File: portfolio/src/main/webapp/images.html [hunk 2]
```diff
@@ -9,19 +9,23 @@
...
-        <a href="images/pic8.jpg"><img src="images/pic8.jpg" alt="Singapore" width="128" height="128"/></a>
-        <a href="images/pic9.jpg"><img src="images/pic9.jpg" alt="France" width="128" height="128"/></a>
-        <a href="images/pic10.jpg"><img src="images/pic10.jpg" alt="van" width="128" height="128"/></a>
-        <a href="images/pic11.jpg"><img src="images/pic11.jpg" alt="van1" width="128" height="128"/></a>
+        <a href="images/pic1.jpg"><img src="images/pic1.jpg" alt="ubc" width="128"/></a>
+        
+        <a href="images/pic3.jpg"><img src="images/pic3.jpg" alt="monkey" width="128"/></a>
+        <a href="images/pic10.jpg"><img src="images/pic10.jpg" alt="van" width="128"/></a>
+        <a href="images/pic11.jpg"><img src="images/pic11.jpg" alt="van1" width="128"/></a>
+        <!-- <a href="images/pic4.jpg"><img src="images/pic4.jpg" alt="pig" width="128" height="128"/></a> -->
```
Comment (EvanJuV) [inline portfolio/src/main/webapp/images.html]: If you're no longer going to use these images, let's remove them instead.

File: portfolio/src/main/webapp/index.html [hunk 3]
```diff
@@ -22,16 +23,49 @@ <h1>Hanze Wang</h1>
...
+        <textarea name="text-input"></textarea>
+        <br/>
+        <input type="submit" />
+        
+      </form>
+
+    
+
+      <h1>Say hello</h1>
+      <!-- <p>Click <a href="/hello">here</a> to view content generated by a servlet.</p> -->
```
Comment (EvanJuV) [inline portfolio/src/main/webapp/index.html:44]: Let's use comments to describe our code, not to hide things.

File: portfolio/src/main/webapp/script.js [hunk 4]
```diff
@@ -39,3 +39,16 @@ function addRandomQuote() {
...
+
+  async function showHello() {
+    const responseFromServer = await fetch('/hello');
+    const textFromResponse = await responseFromServer.json();
+  
+    const helloContainer = document.getElementById('hello-container');
+
+    helloContainer.innerText = textFromResponse[Math.floor(Math.random() * textFromResponse.length)];
+
+    // console.log(textFromResponse.get(0));
```
Comment (EvanJuV) [inline portfolio/src/main/webapp/script.js]: Please also remove these comments.

**Review by EvanJuV — CHANGES_REQUESTED** [review]

**Review by HantzeWANG — COMMENTED** [review]
review changed

File: portfolio/src/main/webapp/index.html (hunk 3 above)
Comment (EvanJuV) [reply portfolio/src/main/webapp/index.html:44]: You forgot to remove this comment.

**Review by EvanJuV — APPROVED** [review]

**Pull request merged**